modules/pam_access/pam_access.c

```diff
@@ -306,6 +306,23 @@ isipaddr (const char *string, int *addr_type,
   return is_ip;
 }
 
+/* is_local_addr - checks if the IP address is local */
+static int
+is_local_addr (const char *string, int addr_type)
+{
+  if (addr_type == AF_INET) {
+    if (strcasecmp(string, "127.0.0.1") == 0) {
+      return YES;
+    }
+  } else if (addr_type == AF_INET6) {
+    if (strcasecmp(string, "::1") == 0) {
+      return YES;
+    }
+  }
+
+  return NO;
+}
```

Comment:
Hmm, why strcasecmp?

Reply:
Given that ip addresses don't contain anything case sensitive, `strcmp` should be enough.